src/App.js

```diff
@@ -1,16 +1,64 @@
 import React from 'react';
+import {
+  BrowserRouter,
+  Routes,
+  Route
+} from "react-router-dom";
```

Comment:
Throughout this file you seem to have used `Routes` when you should have used `Switch` from `react-router-dom`. Try replacing this import and the JSX on lines 46 and 59 below with `Switch`, I expect that will fix the error you posted in the Slack channel.